### PR TITLE
[Django_axes_warning] adding axes beckend and middleware

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- https://django-axes.readthedocs.io/en/latest/2_installation.html

### CODE REVIEW

File: desktop/core/src/desktop/conf.py [hunk 1]
```diff
@@ -1071,7 +1071,7 @@ def get_deprecated_login_lock_out_by_combination_browser_user_agent():
   help=_("Configuration options for user authentication into the web application."),
   members=dict(
     BACKEND=Config("backend",
-                   default=["desktop.auth.backend.AllowFirstUserDjangoBackend"],
+                   default=['axes.backends.AxesBackend', "desktop.auth.backend.AllowFirstUserDjangoBackend"],
```
Comment (romainr) [inline desktop/core/src/desktop/conf.py:1074]: Won't this brake when setting manually, e.g. LDAP, backends?

Can't we auto add it in settings.py?

**Review by romainr — APPROVED** [review]

**Comment by agl29** [comment]
For py3 tests are failing in CI but not locally. I am trying to figure out why locally not failing why CI tests are failing.

File: desktop/libs/dashboard/src/dashboard/tests.py [hunk 2]
```diff
@@ -32,6 +32,7 @@
 from dashboard.facet_builder import _round_number_range
 from dashboard.models import Collection2, augment_response
 from dashboard.controller import DashboardController
+from unittest.mock import Mock
```
Comment (romainr) [inline desktop/libs/dashboard/src/dashboard/tests.py:35]: Will brake depending on the Python, to importer properly

Comment (agl29) [reply desktop/libs/dashboard/src/dashboard/tests.py:35]: As I am not getting errors in py3 locally, so need to test on CI for every commit.
so I am doing testing now will send the final PR if all good.

**Comment by agl29** [comment]
any idea why tests are failing -> https://github.com/cloudera/hue/pull/2297/checks?check_run_id=3045468878 @romainr

**Comment by agl29** [comment]
As some tests are failing in CI -> 

Screenshot 2021-07-16 at 6 26 11 PM


but when I am trying them locally they not failing and tried also ( make clean but same result)

Screenshot 2021-07-16 at 6 27 49 PM

@romainr

**Review by romainr — APPROVED** [review]

**Pull request merged**